varcode/effects/effect_prediction_coding_in_frame.py

```diff
@@ -208,7 +208,7 @@ def predict_in_frame_coding_effect(
         reference_protein_length - aa_mutation_start_offset - 1)
 
     if mutant_codons_contain_stop and (
-            n_aa_alt < n_ref_amino_acids_after_mutated_site):
+            n_aa_alt <= n_ref_amino_acids_after_mutated_site):
```

Comment:
I think that the `mutant_codon` in this case would be "TTT" and thus `mutant_codons_contain_stop` would be False, but maybe I'm wrong.